### ISSUE TITLE
VA Forms System Core [Component]: Multiple screens - Step 1, 6, 7, 8 - Radio buttons and checkboxes need fieldsets and legends

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#45724]
### Point of contact

Angela Fowler

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

On Step 1, 6, 7, 8, several of the radio button and checkbox components aren't contained in fieldsets.
For example, when navigating to the "relationship to deceased" radio button, the question isn't read. Similar issues occur on other steps as well.

### Reproduction steps

1. Go to one of the steps listed.
2. Inspect HTML.
3. Note that the radio button and checkbox question lack fieldsets.

### Proposed solution or next steps

Add the radio button and checkbox question to fieldsets. Like:

```html
<fieldset>
<lenged>Question text</legend>
…radio buttons…
</fieldset>
```

A bigger question? Is this not the default container for this component in the design system? Is that something we could make work out of the box when teams use radio button or checkbox components?

### References, articles, or WCAG support

WCAG 1.3.1 Info and Relationships

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else
